src/watchmaker/workers/salt.py

```diff
@@ -7,195 +7,157 @@
-            bootstrapcmd = [
-                'sh',
-                self.saltbootstrapfilename,
-                '-g',
-                self.config['saltgitrepo']
-            ]
-            if self.config['saltversion']:
-                bootstrapcmd.append('git')
-                bootstrapcmd.append(self.config['saltversion'])
-            else:
-                lslog.debug('No salt version defined in config.')
-            subprocess.call([bootstrapcmd])
+    def _set_salt_dirs(self, srv):
+        self.salt_file_root = os.sep.join((srv, 'states'))
+        self.salt_base_env = os.sep.join((self.salt_file_root, 'base'))
+        self.salt_formula_root = os.sep.join((srv, 'formulas'))
+        self.salt_pillar_root = os.sep.join((srv, 'pillar'))
 
     def _prepare_for_install(self):
-
         if self.config['formulastoinclude']:
-            self.formulastoinclude = self.config['formulastoinclude']
+            self.formulas_to_include = self.config['formulastoinclude']
 
         if self.config['formulaterminationstrings']:
-            self.formulaterminationstrings = self.config[
+            self.formula_termination_strings = self.config[
                 'formulaterminationstrings']
 
-        self.sourceiss3bucket = self.config['sourceiss3bucket']
-        self.computername = self.config['computername']
-        self.entenv = self.config['entenv']
-        self.create_working_dir('/usr/tmp/', 'saltinstall')
+        self.computer_name = self.config['computername']
+        self.ent_env = self.config['entenv']
+        if not self.is_s3_bucket:
+            self.is_s3_bucket = self.config['sourceiss3bucket']
+
+        self.create_working_dir(
+            self.salt_working_dir,
+            self.salt_working_dir_prefix
+        )
 
         self.salt_results_logfile = self.config['salt_results_log'] or \
-            os.sep.join((self.workingdir, 'saltcall.results.log'))
+            os.sep.join((self.working_dir, 'salt_call.results.log'))
 
         self.salt_debug_logfile = self.config['salt_debug_log'] or \
-            os.sep.join((self.workingdir, 'saltcall.debug.log'))
+            os.sep.join((self.working_dir, 'salt_call.debug.log'))
 
-        self.saltcall_arguments = [
+        self.salt_call_args = [
             '--out', 'yaml', '--out-file', self.salt_results_logfile,
             '--return', 'local', '--log-file', self.salt_debug_logfile,
             '--log-file-level', 'debug'
         ]
 
-        for saltdir in [self.saltfileroot,
-                        self.saltbaseenv,
-                        self.saltformularoot]:
+        for salt_dir in [
+            self.salt_file_root,
+            self.salt_base_env,
+            self.salt_formula_root
+        ]:
             try:
-                os.makedirs(saltdir)
+                os.makedirs(salt_dir)
             except OSError:
-                if not os.path.isdir(saltdir):
+                if not os.path.isdir(salt_dir):
                     raise
```

Comment:
naked raise should be cleaned up. this i think was my bad, as we hadnt had a clear path on exception handling, unless its always being caught.